YAXLib/YAXSerializer.cs

```diff
@@ -2833,7 +2833,7 @@ private IEnumerable<MemberWrapper> GetFieldsToBeSerialized(UdtWrapper typeWrappe
         /// <returns>the sequence of fields to be serialized for the serializer's underlying type.</returns>
         private IEnumerable<MemberWrapper> GetFieldsToBeSerialized()
         {
-            return GetFieldsToBeSerialized(m_udtWrapper);
+            return GetFieldsToBeSerialized(m_udtWrapper).OrderBy(t=>t.Order);

```

Comment:
I'm wondering will it retain the order of properties when the attribute is not specified. I don't want to see stuff serialized in random order simply because all the orders are 0. Need to find a way to use the property definition order as the tie breaker.